.github/workflows/python-lint.yml

```diff
@@ -0,0 +1,32 @@
+name: "Python Linting"
+
+on:
+  pull_request:
+    types: ["opened", "synchronize", "reopened", "edited"]
+  release:
+    types: published
+  workflow_dispatch:
+
+jobs:
+  flake8_py3:
+    name: "Linting with Flake8"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: "Set up Python"
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+
+      - name: "Install flake8"
+        run: pip install flake8
+
+      - name: "Install flake8-django"
+        run: pip install flake8-django
+
+      - name: "Run flake8"
+        uses: liskin/gh-problem-matcher-wrap@v1
+        with:
+          linters: flake8
+          run: flake8 --max-line-length 150 server/
```

Comment:
i usually use flake8 and black together for my python projects but im not sure if they are commonly used together in django